slave/centos-7/manifest.yml

```diff
@@ -1,6 +1,6 @@
 images:
 - name: buildslave-centos-7-zfs-head
-  base-ami: ami-c7d092f7
+  base-ami: ami-edffe8dd
```

Comment:
This change doesn't look correct. This should point to the AMI of the instance used to build the image. (So, in this case https://aws.amazon.com/marketplace/pp/B00O7WM7QW which is the official CentOS image got from following links on https://www.centos.org/download/).